docs/core/docker/publish-as-container.md

```diff
@@ -182,6 +182,46 @@ You can control many aspects of the generated container through MSBuild properti
 > [!NOTE]
 > The only exceptions to this are `RUN` commands. Due to the way containers are built, those cannot be emulated. If you need this functionality, you'll need to use a _Dockerfile_ to build your container images.
 
+### Container image naming configuration
+
+Container images follow a specific naming convention. The name of the image is composed of several parts, the registry, optional port, repository, and optional tag and family.
+
+```dockerfile
+REGISTRY[:PORT]/REPOSITORY[:TAG(S)[-FAMILY]]
```

Comment:
you use TAG/TAGS below in a few places - we should be clear that if you specify `ContainerImageTags` you get _multiple_ generated containers, one for each tag. In no case do you get an image like `mcr.microsoft.com/dotnet/sdk:8.0;latest-alpine` as one thing - instead you'd get

* mcr.microsoft.com/dotnet/sdk:8.0-alpine
* mcr.microsoft.com/dotnet/sdk:latest-alpine

Reply:
I've made this more clear...